dojo/tools/wazuh/parser.py

```diff
@@ -18,73 +25,59 @@ def get_label_for_scan_types(self, scan_type):
     def get_description_for_scan_types(self, scan_type):
         return "Wazuh"
 
-    def get_findings(self, filename, test):
-        data = json.load(filename)
+    def get_findings(self, file, test):
+        data = json.load(file)
+
+        if not data:
+            return []
+
         # Detect duplications
         dupes = dict()
 
-        try:
-            vulnerability = data[next(iter(data.keys()))]["affected_items"]
-        except (KeyError, StopIteration):
-            return list()
-
-        if vulnerability is None:
-            return list()
+        # Loop through each element in the list
+        for entry in data:
+            vulnerabilities = entry.get("data", {}).get("affected_items", [])
+            for item in vulnerabilities:
+                if (
+                    item["condition"] != "Package unfixed"
+                    and item["severity"] != "Untriaged"
+                ):
+                    id = item.get("cve")
+                    package_name = item.get("name")
+                    package_version = item.get("version")
+                    description = item.get("condition")
+                    severity = item.get("severity").capitalize()
+                    agent_ip = item.get("agent_ip")
+                    links = item.get("external_references")
 
```

Comment:
also add cvssv3_score and pusblish__date

`cvssv3_score = item.get("cvss3_score")

publish_date = item.get("published")`